### PR TITLE
Update credits

### PR DESCRIPTION
**Pull request by caseyedavis12**

### CODE REVIEW

**Comment by mccalluc** [comment]
(I'll sort this out: Need to clear up the conflicts, take screen shots, and then remove the logos until we actually have permission.)

File: app/views/override/credits/credits-advocates.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Credits: The Advocates
+  
+[The Advocates](/collections/advocates-advocates/full-program-video) collection was made possible by a grant from the Open Society Foundations (OSF).
+
+<a href="http://www.opensocietyfoundations.org"><img height="60" src="https://s3.amazonaws.com/openvault.wgbh.org/logos/OSF.jpg"
```
Comment (mccalluc) [inline app/views/override/credits/credits-advocates.md]: @caseyedavis12 : Unless there is a compelling reason, I would prefer to minimize the usage of html in md. (Though in this case, along with all the other outbound links, I think we could add a `target="_blank"`, and that would be our compelling reason.)

**Comment by mccalluc** [comment]
(just rebased and pushed back to origin...)

**Comment by mccalluc** [comment]
@caseyedavis12 : Duplicated text in italics. Is this really what you want?
http://localhost:3000/credits:
>
*The creation of Open Vault was made possible through a grant from the Institute of Museum and Library Services (IMLS). The views, findings, conclusions or recommendations expressed in this website do not necessarily represent those of the* Institute of Museum and Library Services.
>
*The prototype website was funded in part by The John D. and Catherine T. MacArthur Foundation.*
>
This site brings together materials from three previous web sites
>
>    New Television Workshop
>    Say Brother
>    Ten O’Clock News
>
As well as new material from War and Peace in the Nuclear Age
>
*IMLS further funded the development of Open Vault in 2008 by supporting the Vietnam Collection.*
>
Subsequent funding by The Andrew W. Mellon Foundation helped inform the later design and additional functionality of Open Vault, as well as the scholar exhibits.

http://localhost:3000/credits/credits-open-vault:
>
*The creation of Open Vault: WGBH Media Library and Archives Web site was made possible through a grant from the Institute of Museum and Library Services (IMLS). Any views, findings, conclusions or recommendations expressed in this website do not necessarily represent those of the* IMLS.
>
*The prototype website was funded in part by The John D. and Catherine T. MacArthur Foundation.*
>
*IMLS further funded the development of Open Vault in 2008 by supporting the Vietnam Collection.*
>
All content on this site is the intellectual property of WGBH Educational Foundation, which is solely responsible for this work.

File: app/views/override/credits/credits-open-vault-research.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ facilitates research access to the collections. Further, working with Ithaka
 S+R, we investigated the sustainability of online archives for academic use. 
 Beyond Open Vault Research, lessons from both Phase One and Phase Two of the 
 Digital Library Project also informed the design and development of Open Vault 
-Vietnam.
+Vietnam. In the third phase, we explored the best methods for bringing our catalog online for scholarly and public access and helped drive new interest and scholarly work around our resources.
```
Comment (mccalluc) [inline app/views/override/credits/credits-open-vault-research.md]: @caseyedavis12 : s/helped drive/driving/ ?

File: app/views/override/credits/credits-mow.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Credits: March on Washington
+  
+The [March on Washington](/collections/march-march-on-washington) collection was supported in part by Save America's Treasures through a partnership between the National Endowment for the Arts and the National Park Service, Department of the Interior.
+
+<a href="http://www.arts.gov"><img height="80" src="https://s3.amazonaws.com/openvault.wgbh.org/logos/NEA.jpg"
```
Comment (mccalluc) [inline app/views/override/credits/credits-mow.md]: @caseyedavis12 : wrong logo?

Comment (caseyedavis12) [inline app/views/override/credits/credits-mow.md]: @mccalluc: This is the right logo. National Endowment for the Arts Saving America's Treasure's program.

File: app/views/override/credits/credits-open-vault-research.md (hunk 2 above)
Comment (caseyedavis12) [inline app/views/override/credits/credits-open-vault-research.md]: @mccalluc: I don't think it requires an edit.

**Comment by caseyedavis12** [comment]
@mccalluc: Yep, keep as is.

**Comment by mccalluc** [comment]
@caseyedavis12:  Here are your screen shots. I'll now comment out the logos.

screen shot 2016-03-11 at 11 09 51 am
screen shot 2016-03-11 at 11 09 43 am
screen shot 2016-03-11 at 11 09 35 am
screen shot 2016-03-11 at 11 09 26 am
screen shot 2016-03-11 at 11 09 17 am
screen shot 2016-03-11 at 11 09 06 am
screen shot 2016-03-11 at 11 08 53 am
screen shot 2016-03-11 at 11 08 37 am
screen shot 2016-03-11 at 11 08 23 am
screen shot 2016-03-11 at 11 08 08 am
screen shot 2016-03-11 at 11 07 54 am
screen shot 2016-03-11 at 11 07 09 am

**Comment by caseyedavis12** [comment]
@mccalluc: Thank you very much.

**Pull request merged**